### PR TITLE
Add postalignqc output to prp

### PR DESCRIPTION
**Pull request by ryanjameskennedy**
# Description
_Summary of the changes made:_
* `post_align_qc` output added to prp output

## Primary function of PR
- [ ] Hotfix
- [x] Minor functionality improvement
- [ ] Major functionality improvement / New type of analysis
- [ ] Backward-breaking functionality

# Testing
Tested using the output from `post_align_qc` output

### CODE REVIEW

**Review by sylvinite — APPROVED** [review]
Looks good!

File: bin/pipeline_result_processor/prp/cli.py [hunk 1]
```diff
@@ -55,6 +56,7 @@ def cli():
 @click.option("-c", "--cgmlst", type=click.File(), help="cgMLST prediction results")
 @click.option("-v", "--virulence", type=click.File(), help="Virulence factor prediction results")
 @click.option("-r", "--resistance", type=click.File(), help="resfinder resistance prediction results")
+@click.option("--quality", type=click.File(), help="postalignqc qc results")
```
[the remark this answers is not in the document]
Comment (ryanjameskennedy) [reply bin/pipeline_result_processor/prp/cli.py:59]: Already used by `quast` maybe `-b`?

[the remark this answers is not in the document]
Comment (sylvinite) [reply bin/pipeline_result_processor/prp/cli.py:59]: Yea true. -p is taken too :<. -b sounds fine. Authors discretion

[the remark this answers is not in the document]
Comment (ryanjameskennedy) [reply bin/pipeline_result_processor/prp/cli.py]: Made a plan for `-p`

**Pull request merged**